### ISSUE TITLE
Just a few feature ideas

### ISSUE THREAD

**Issue by jojomoore2007** [nothjarnan/axiom#43]
I have a few ideas. 

1. First off, the startup file shouldn't just error when it can't find sys.lua, but instead, it should reinstall it.

1. Also, you should really have support for movable windows. And no, I'm not asking for real-time manipulation of actual window parts moving based on you dragging it, but rather, when the user clicks a "Move" option, perhaps in a right-click/context menu, then it makes the window move to the X and Y position of the mouse next time the user clicks, and redraws all windows.

1. Possibly the ability to actually CLOSE the file manager rather than trying to hide it? Yes, I noticed this. The close button will attempt to hide the window and redraw the desktop over it, but because of the window being "active", it will unhide itself, making it impossible to close. This is a major bug, making it impossible to see the desktop.

1. DON'T CRASH AXIOM WHEN THE STARTUP FILE IS CHANGED!!!!! This makes it impossible to save or recover your files!

1. The ability to boot from a safe-mode disk! This is perfect for when you can't boot Axiom and CraftOS is impossible to reach because of a crash shutting down the computer! This would also be useful if you got any malware/hijacks/trojans on Axiom by accident and needed to reinstall Axiom without touching any of your files!

Thanks!

-Joey (TheQWERTYCoder)

P.S.: Can someone post in the forums that reCAPTCHA V1 is shut down? I recently found ComputerCraft and can't make an account due to this issue.

**Comment by nothjarnan** [comment]
I agree with your points, except the moveable window bit. The OS was never built with windowing in mind, only my latest project currently has support for it. And for #5, you can insert the computer into a disk drive to tamper with the files.

Just a note though, this is a stale project with no real intention to continue unless a feature becomes deprecated via CC updates, and as for the CC forum captcha, we've done everything we can.

**Comment by jojomoore2007** [comment]
For #5: What if you forget to set the label?

**Comment by LDDestroier** [comment]
Axiom automatically sets the computer label upon installation, so that won't be an issue.

**Comment by jojomoore2007** [comment]
K.
Thanks!

**Comment by jojomoore2007** [comment]
Hey,

I just signed the petition "Loudoun county public schools: No school day
after halloween for loudoun county public schools" and wanted to see if you
could help by adding your name.

Our goal is to reach 10,000 signatures and we need more support. You can
read more and sign the petition here:

http://chng.it/5gY4ctKJnt

Thanks!
Joey

**Comment by QuickMuffin8782** [comment]
GitGet is not working... and so is the installer on CC: Tweaked. I am using CCEmuX
